modules/cnv-web-console-overview-dashboard.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id="web-console-overview-dashboard_{context}"]
+= Overview Dashboard
+
+* *Details* card provides an brief overview of informational cluster details, including:
```

Comment:
"an brief" Typo

Reply:
Not having 'The' before the card name makes it weird to read. Applies to all.

Reply:
Thanks Vikram, corrected typo. 

Ashley suggested not saying "card" numerous times, so I added an intro sentence and eliminated the articles and "card":

_The overview dashboard consists of the following cards:_

To me, it looks like if you add the article you also have to add "card". For example, "The Details card" not "The Details". 

Your call @aburdenthehand.

Reply:
Much improved.

Reply:
Thanks. Done.